src/js/helpers/MapGraphics.ts

```diff
@@ -32,16 +34,44 @@ export function setNewGraphic({
     });
   }
 
-  const polygonOrPointSymbol = (type: string): SimpleFillSymbol =>
-    type === 'polygon' ? getCustomSymbol() : getImagerySymbol();
+  const setSymbol = (symbolType: string): any => {
+    switch (symbolType) {
+      case 'polygon':
+        return getCustomSymbol();
+      case 'point':
+        return getPointSymbol();
+    }
+  };
+
+  const setGeometry = (symbolType: string, geometry: __esri.Geometry): any => {
+    switch (symbolType) {
+      case 'polygon':
+        return new Polygon(geometry);
+      case 'point':
+        return new Point(geometry);
+    }
```

Comment:
if we have a switch statement, lets add default case too. it can try and return a Polygon?